nerdlandbot/commands/kerk.py

```diff
@@ -0,0 +1,30 @@
+import typing
+import asyncio
+import discord
+from discord.ext import commands
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+from nerdlandbot.helpers.parser import parse_channel
+
+from datetime import datetime
+
+class Kerk(commands.Cog, name="kerk"):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="16ukerk", hidden=True)
+    async def cmd_kerk16(self, ctx: commands.Context, mention:typing.Optional[str] = None, *, message:typing.Optional[str] = None):
+        #check if message in lounge
+        if ctx.channel.name == ("lounge"):
+            #check if user mentioned  
+            if mention and mention[:2] == "<@":
+                #set a task to check if it's 16:00 and posts message
+                while not (datetime.now().hour == 16 and datetime.now().minute == 00):
+                    await asyncio.sleep(59)
```

Comment:
This should probably be changed to a Scheduler that runs every 24 hours. This scheduler can have a list to which new reminders need to be appended. Every time the scheduler runs, it calculates how long until 16:00, sleeps for that amount of time and then it atomically gets and clears the list, mentioning everyone in that list (one by one or together).